package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "eve-operations-suite",
+  "version": "0.0.1",
+  "author": "Nate Reinhardt",
+  "description": "Suite of tools to manage operations for Eve Online",
+  "scripts": {
+    "start": "concurrent \"npm run tsc:w\" \"npm run lite\" ",
+    "tsc": "tsc",
+    "tsc:w": "tsc -w",
+    "lite": "lite-server",
+    "typings": "typings",
+    "postinstall": "typings install"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "angular2": "2.0.0-beta.9",
+    "systemjs": "0.19.24",
+    "es6-promise": "^3.0.2",
+    "es6-shim": "^0.33.3",
+    "reflect-metadata": "0.1.2",
+    "rxjs": "5.0.0-beta.2",
+    "zone.js": "0.5.15",
+    "bootstrap": "^3.3.6"
+  },
+  "devDependencies": {
+    "concurrently": "^2.0.0",
+    "lite-server": "^2.1.0",
+    "typescript": "^1.8.9",
+    "typings": "^0.7.9"
+  },
+  "repository": {}

```

Comment:
I will point that to this repo once i get around to it. Not sure what is exactly used for so I left it there to come back to it.